templates/components/set-password-migration-form.jsx

```diff
@@ -0,0 +1,42 @@
+var React = require('react');
+var validators = require('../lib/validatorset');
+var Form = require('./form/form.jsx');
+var IconText = require('./icontext.jsx');
+
+var fields = [
+  {
+    'password': {
+      'placeholder': 'Enter your new password',
+      'type': 'text',
+      'validator': 'username',
+      'errorMessage': 'Invalid username'
+    }
+  }
+];
+
+var fieldsValidators = validators.getValidatorSet(fields);
+
+var SetPasswordMigration = React.createClass({
+  render: function() {
+    return (
+      <div className="migrateKeyContainer">
+        <IconText
+          iconClass="emailSentIcon fa fa-lock"
+          className="emailSent arrow_box">
+            <p className="emailSentHeader">Set your password</p>
```

Comment:
It needs to be a proper header where I have control over the line height and spacing. Not sure if the IconText is uses the same... I have tried the other day and thought each page doesn't seem to have the same layout and I ended up using this.